python/revng/daemon/static_handlers.py

```diff
@@ -128,6 +129,15 @@ async def resolve_targets(_, info, *, pathspec: str):
     return result
 
 
+@query.field("target")
+async def resolve_target(_, info, *, step: str, container: str, target: str) -> Optional[Target]:
+    manager: Manager = info.context["manager"]
+    result = await run_in_executor(
+        lambda: manager.deserialize_target(f"{step}/{container}/{target}")
+    )
+    return result.as_dict() if result is not None else None
```

Comment:
Not sure what this does.

Reply:
It's the handler to retrieve a single target, I use it to detect if a binary has been uploaded by doing:
```
{
  target(step: "begin", container: "input", target: ":Binary") {
    ready
  }
}
```
if ready is `true` then the target exists otherwise I get `target: null`